common/models/scaler_run_info.go

```diff
@@ -1,11 +1,11 @@
 package models
 
 type ScalerRunInfo struct {
```

Comment:
[nit] I thought we were removing this?

Reply:
Ooh, I didn't catch this struct. I removed the usage of the struct in `environment_scaler.go` over in #440 PR, but not the struct. I'll fix that in #440.